tox.ini

```diff
@@ -2,7 +2,7 @@
 
 [tox]
 # Also see .travis.yml where this is maintained separately.
-envlist=py26,py27,py34,docs
+envlist=py26,py27,py34,py35,docs
```

Comment:
good idea. You need to also activate this [for Travis](https://github.com/catlee/mohawk/blob/bewit/.travis.yml#L10).